.github/workflows/ci.yml

```diff
@@ -120,17 +123,68 @@ jobs:
         run: |
           ddev composer install
           ddev pull platform -y
-          ddev drush deploy -y
+
+      - name: Run phpstan
+        run: ddev php ./vendor/bin/grumphp run --tasks=phpstan --no-interaction
+
+      - name: Rebuild cache
+        run: |
+          ddev drush cache:rebuild -y
+          ddev drush config:status
+
+      - name: Import Config and Run updates
+        run: |
+          ddev drush updatedb --no-cache-clear --yes
+          ddev drush config:import --yes
+          ddev drush cache:rebuild --yes
+          ddev drush deploy:hook --yes
+
+      # Validate config changes
+      - name: Validate Config Export
+        id: validate_config
+        run: |
+          status_output=$(ddev drush config:status 2>&1)
+          echo "$status_output"
+          echo "status_output<<EOF" >> $GITHUB_OUTPUT
+          echo "$status_output" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
```

Comment:
Some comments might be helpful here. We will come back looking at this change in the future, I'm sure.